README.md

```diff
@@ -68,7 +69,8 @@ Once the project is built, the result "`opus.rtf`" will end up in the "`product`
 You can change the output directory by using the "`PRODUCT_DIRECTORY`" argument:
```

Comment:
This here fails for me using https://github.com/compor/cmake-pandocology:

```cmake
list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake-helper/cmake-pandocology")
include(pandocology)

add_document(
    TARGET readme
    OUTPUT_FILE README.html
    SOURCES ./README.md
)
```

Error message:
```
CMake Error at cmake-helper/cmake-pandocology/pandocology.cmake:258 (add_custom_command):
  add_custom_command Wrong syntax.  A TARGET or OUTPUT must be specified.
Call Stack (most recent call first):
  CMakeLists.txt:93 (add_document)


CMake Error at cmake-helper/cmake-pandocology/pandocology.cmake:330 (add_custom_command):
  add_custom_command Wrong syntax.  A TARGET and OUTPUT can not both be
  specified.
Call Stack (most recent call first):
  CMakeLists.txt:93 (add_document)
```

Reply:
I haven't checked anything locally yet, but from a quick glance online, it seems that your submodule requires an update to the latest code of this PR judging from the commit hash alone.